node-process-argv/echo.js

```diff
@@ -0,0 +1,6 @@
+
+// console.log("Command line arguements!")
+
+console.log("Hello, computer!")
```

Comment:
You should be echoing out what the user input to start the process. For example if someone typed `node echo.js "This is Fred"` then the file should print out `This is Fred`. The data will be stored in the `process.argv` array